recipes/g2o/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "g2o" %}
+{% set version = "2019.11.23" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/RainerKuemmerle/g2o/archive/fcba4eaca6f20d9a5792404cc8ef303aeb8ba5d2.tar.gz
+    sha256: 8424d593e12c3032ead50c782fd2a55c62be5110143bf55378b763d374ac0ebd
+
+build:
+  number: 0
+  skip: true  # [osx or win and vc<14]
+  run_exports:
+    - {{ pin_subpackage(name, exact=True) }}
```

Comment:
Exact would have the build number as well

Reply:
@isuruf Thanks. I changed it to `max_pin`.